<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMainWindow</class>
 <widget class="QWidget" name="frmMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>268</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Orange Killer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>OrangeKill.png</normaloff>OrangeKill.png</iconset>
  </property>
  <widget class="QGroupBox" name="GrpBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>601</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Orange Killer</string>
   </property>
   <widget class="QLabel" name="lblHOST">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>38</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP ADDR :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblID">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>71</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPW">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>103</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PASSWORD :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPATH1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>137</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>첫번째 경로 :</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtIPADDR">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtHOSTID">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>63</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtPATH1ST">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>129</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lblPATH2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>169</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>두번째 경로 :</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtPATH2ND">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>162</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnAction">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>예토전생</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lttHOSTPW">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>96</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkRepeat">
    <property name="geometry">
     <rect>
      <x>392</x>
      <y>36</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>오렌지 5분마다 반복제거</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkLocal">
    <property name="geometry">
     <rect>
      <x>392</x>
      <y>60</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>로컬영역에서 실행</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
